lib/tasks/state.rake

```diff
@@ -0,0 +1,16 @@
+namespace :state do
+  desc "Migrate state from the state property to Sipity"
+  task migrate: :environment do
+    ['MultiVolumeWork', 'ScannedResource', 'ImageWork', 'RasterWork', 'VectorWork'].each do |model|
+      ActiveFedora::SolrService.get("has_model_ssim:#{model}")['response']['docs'].each do |doc|
```

Comment:
👍  — I also noticed that I had `book_works` hardcoded, so I replaced that with a hash of workflows to models they contain.